Workflow.cpp

```diff
@@ -161,7 +164,37 @@ void Workflow::inlineMethodCalls(AnalysisContextRef ac)
                 log->LogInfo("Structures analyzed in %lu ms", elapsed.count());
 
                 InfoHandler::applyInfoToView(info, bv);
-            } catch (...) {
+
+				const auto msgSendFunctions = findMsgSendFunctions(bv);
+				for (auto addr : msgSendFunctions)
+				{
+					BinaryNinja::QualifiedNameAndType nameAndType;
+					std::string errors;
+					std::set<BinaryNinja::QualifiedName> typesAllowRedefinition;
+
+					// void *
+					auto retType = BinaryNinja::Type::PointerType(bv->GetAddressSize(), BinaryNinja::Type::VoidType());
```

Comment:
be careful when assigning things to `void*` you should probably set the confidence to 0